src/style.css

```
@@ -1 +1,100 @@
-/* add styles here */
+@font-face {
+  font-family: AvenirBook;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 16px;
+  font-weight: 300;
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+.sch-bar {
+  position: relative;
+  margin-top: 20px;
+}
+
+.sch-bar:hover {
+  border-radius: 4px;
+  background: #fff;
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.20);;
+}
+
+.sch-bar__input {
+  width: 100%;
+  position: relative;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-sizing: border-box;
```

you have already added box-sizing to this block `* {}`, so it's redundant here
```suggestion
  box-sizing: border-box;
```